components/match2/wikis/rainbowsix/match_group_input_custom.lua

```diff
@@ -317,10 +319,17 @@ function mapFunctions.getExtraData(map)
 		t1firstside = {rt = map.t1firstside, ot = map.t1firstsideot},
 		t1halfs = {atk = map.t1atk, def = map.t1def, otatk = map.t1otatk, otdef = map.t1otdef},
 		t2halfs = {atk = map.t2atk, def = map.t2def, otatk = map.t2otatk, otdef = map.t2otdef},
-		t1bans = {map.t1ban1, map.t1ban2},
-		t2bans = {map.t2ban1, map.t2ban2},
+		t1bans = {},
+		t2bans = {},
 		pick = map.pick
 	}
+
+	for opponentIndex = 1, MAX_NUM_OPPONENTS do
+		for banIndex = 1, MAX_NUM_BANS do
+			local ban = map['t' .. opponentIndex .. 'ban' .. banIndex]
+			table.insert(map.extradata['t' .. opponentIndex .. 'bans'], CharacterNames[ban and ban:lower()])
```

Comment:
mind if tXban1 is nil while tXban2 is not you will get a different table with this
{nil, tXban2} vs {tXban2}

Reply:
Summary iterates bans using [ipairs](https://github.com/Liquipedia/Lua-Modules/blob/9f3dfe2cc13d75f1a11c0143f385722729b5b949/components/match2/wikis/rainbowsix/match_summary.lua#L522), so if tXban1 is missing, tXban2 isn't displayed.

Reply:
just making aware of it^^

Reply:
Yes, thank you. Seems like operator order matter, so it can't be nil :D